### ISSUE TITLE
Game closing

### ISSUE THREAD

**Issue by Whopperman48** [Uuvana-Studios/longvinter-windows-client#240]
I have seen this on some other peoples posts, when I join a server, I can play, get looted, and when I look for a place to place a tent I get kicked. I have tried 2 separate servers multiple times and it kicks me out around 15-30 min into play and resets all my progress and I have to start over from scratch. There is no error message, but it brings me back to the update list when you first launch the game.

**Comment by Morgoth1310** [comment]
yup same. unexpectedly kicks me to main screen and then servers don't load.
then when I restart the game all progress is lost

**Comment by Ultimodrew** [comment]
The same is happening to me since I got the game. And what makes it worse, is the over all map is so small that people have already gathered up all the viable space to put a tent down, which I believe is what is needed to save your progress. I have tried several servers and most instantly fail to load. The ones that do, I get the 30-40 mins of game play up to the point I am looking for a spot to place my tent and get kicked. I get this game is still in Beta, but it is utterly ridiculous to be charged what I assume is full price for a game that is not ready.
